### PR TITLE
Update to use official iTwin.js 3.0 release

### PR DESCRIPTION
**Pull request by aruniverse**

### CODE REVIEW

**Comment by aruniverse** [comment]
@austeja-bentley please review the version-policy and rush.json file. I was surprised to see i wasn't prompted for what kind of version bump. Idk if the next bump should be a pre-release , but you guys can probably move away from being in lockstep now

**Comment by calebmshafer** [comment]
@aruniverse I think this repo isn't directly publishing via Rush. They do a publish pack similar to the way core does.

File: common/config/rush/version-policies.json [hunk 1]
```diff
@@ -1,8 +1,6 @@
 [
   {
-    "policyName": "prerelease-monorepo-lockStep",
-    "definitionName": "lockStepVersion",
-    "version": "1.0.0",
-    "nextBump": "prerelease"
+    "policyName": "individual-versioning",
+    "definitionName": "individualVersion"
```
Comment (calebmshafer) [inline common/config/rush/version-policies.json:4]: mind reverting these changes? This is going to break the rush publish.

Comment (aruniverse) [reply common/config/rush/version-policies.json:4]: do i change the version to 1.0.1 or remove the next bump is a pre-release?

Comment (calebmshafer) [reply common/config/rush/version-policies.json]: just updated them. It should be good to go now.

**Review by austeja-bentley — APPROVED** [review]

**Pull request merged**